client/src/actions/actionTypes.js

```diff
@@ -0,0 +1,11 @@
+export const LOAD_MOVIES_SUCCESS = 'LOAD_MOVIES_SUCCESS';
```

Comment:
It makes sense to group all the store related folders.
E.x. you can have a src/store folder and inside it have index.js as a store main file + /actions and /reducers folders there too.